LayoutTests/media/now-playing-webaudio.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Testing basic video exchange from offerer to receiver</title>
+        <script src="../resources/testharness.js"></script>
+        <script src="../resources/testharnessreport.js"></script>
+    </head>
+    <body>
+        <script>
+function waitFor(delay)
```

Comment:
that's in video-test.js already

Reply:
video-test.js is not test harness based, so I prefer not mixing things.